test/DynamoCoreTests/Settings.cs

```diff
@@ -84,6 +85,31 @@ public void SetPythonTemplateFromConfigWithInvalidPath()
             Assert.AreEqual(model.PreferenceSettings.PythonTemplateFilePath, string.Empty);
         }
 
-    }
+        [Test]
+        public void RunSettingsDisableAndEnableRun()
+        {
+            string openPath = Path.Combine(TestDirectory, @"core\RunSettings.dyn");
+            CurrentDynamoModel.OpenFileFromPath(openPath);
+
+            var ws = CurrentDynamoModel.CurrentWorkspace as HomeWorkspaceModel;
+            Assert.AreEqual(0, ws.EvaluationCount);
+
+            // Still in Manual mode so will not run
+            ws.RequestRun();
+            Assert.AreEqual(0, ws.EvaluationCount);
 
+            // Setting the run type to automatic
+            ws.RunSettings.RunType = RunType.Automatic;
```

Comment:
hmm Shouldn't updating RunType trigger a new run?

Reply:
This will not trigger the run() function when the runtype is changed in a dynamo model test as there is no handler present.